src/test/rpc/LedgerRPC_test.cpp

```diff
@@ -652,6 +652,28 @@ class LedgerRPC_test : public beast::unit_test::suite
 
     }
 
+    void testLedgerAccountsType()
+    {
+        testcase("Ledger Request, Accounts Option");
+        using namespace test::jtx;
+
+        Env env {*this};
+
+        env.close();
+
+        Json::Value jvParams;
+        jvParams[jss::ledger_index] = 3u;
+        jvParams[jss::accounts] = true;
+        jvParams[jss::expand] = true;
```

Comment:
might be worth adding a variation with`expand=false` just to make sure the output is what we expect (you could re-use `jvParams` and just change the one param).